docs/build-with-0g/serving-network/provider.md

```diff
@@ -0,0 +1,130 @@
+---
+id: provider
+title: Provider
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+To integrate a service into the 0G Serving Network, you need to transform your service into a **verifiable service** and connect it through the **provider broker container**.
+
+## Verifiable Services
+
+### Service Interface Requirements
+
+Your service need to adhere to the [OpenAI API Interface Standards](https://platform.openai.com/docs/api-reference/chat).
+
+### Verification Interfaces
+
+To ensure the integrity and trustworthiness of services, different verification mechanisms are employed. Each mechanism comes with its own specific set of protocols and requirements to ensure service verification and security.
+
+<Tabs>
+<TabItem value="tee" label="TEE" default>
+
+For TEE (Trusted Execution Environment) verification, when a service starts, it should generate a signing key within the TEE. We require CPU and GPU attestations to ensure the service is running in a Confidential VM with an NVIDIA GPU in TEE mode. These attestations should include the public key of the signing key, verifying its creation within the TEE. All inference results must be signed with this signing key.
+
```

Comment:
Can we add OPML/SPML *coming soon* tags here? Wouldn't want us to solely be classed into TEE ML bucket.

Reply:
Added  

![image](https://github.com/user-attachments/assets/15013f1a-9e07-41a6-a54d-47497292e81a)